tests/test_tiatoolbox.py

```diff
@@ -543,17 +1080,19 @@ def test_command_line_slide_info(_response_all_wsis):
     assert slide_info_result.exit_code == 0
 
 
-def test_command_line_read_region(_response_ndpi, tmp_path):
-    """Test OpenSlide read_region CLI."""
+def test_command_line_read_bounds(_response_ndpi, tmp_path):
```

Comment:
in cli shall we not just keep it read_region, it makes more sense than using read_bounds in cli.

Reply:
Do you want to change the arguments then? It currently accepts four arguments for the bounds:  start_w, end_w, start_h, end_h (left, right, top, bottom)